### PR TITLE
Removes converstion and deconversion from Cult. Cult is now a fixed 9 man team that needs to do 9 sacrifices to summon Nar'sie.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request
Cults now spawn with 9 members, no more no less.
Cultists now cannot convert crewmembers or be deconverted.
Sacrifices do not provide soul shards nor can construct shells be built, summoned, or Artificer created.
Constructs are only obtainable via using Twisting Aura on a live cyborg.
Sacrifices must be living sentient humanoids, or for the body to be the Sacrifice Target.
Sacrifices and Nar'Sie summoning now require EITHER 3 live cultists, or however many cultists are still alive.
This means if you only have 1 or 2 cultists, you can sacrifice with 1 or 2 cultists present.
This also applies to Nar'Sie summoning and the 9 person requirement.
Summoning Nar'Sie now requires that you have 9 sacrifices total on top of the existing sacrifice target objective.
The Bloody Bastard Sword no longer harvests souls into soulstones.

## Conversion modes suck and we banned them officially
Cult as a conversion mode is an absolute mess to get into and 90% of cult converts just go murderboning when they get converted. It does not follow the rule we set out that a new player inducted into a conversion team must know immediately what they need to do, and thus it's been removed from being a conversion gamemode.

## Cult hasn't seen a big shakeup in half a decade
Cult is a stagnant gamemode that a lot of our players don't enjoy because it's stuck in a very unfun cycle of being both a conversion mode and a stagnant mode at the same time. Unshackling it from conversion allows us to explore more design space with the gamemode without fear of the conversion/deconversion aspect causing the design to snowball out of control.

These changes should freshen up the mode and make it significantly more fun to play as, against, and around.
## Changelog
:cl:
balance: Cults now spawn with 9 members, no more no less.
balance: Cultists now cannot convert crewmembers or be deconverted.
balance: Sacrifices do not provide soul shards nor can construct shells be built, summoned, or Artificer created.
balance: Constructs are only obtainable via using Twisting Aura on a live cyborg.
balance: Sacrifices must be living sentient humanoids, or for the body to be the Sacrifice Target.
balance: Sacrifices and Nar'Sie summoning now require EITHER 3 live cultists, or however many cultists are still alive.
balance: This means if you only have 1 or 2 cultists, you can sacrifice with 1 or 2 cultists present.
balance: This also applies to Nar'Sie summoning and the 9 person requirement.
balance: Summoning Nar'Sie now requires that you have 9 sacrifices total on top of the existing sacrifice target objective.
balance: The Bloody Bastard Sword no longer harvests souls into soulstones.
/:cl:

### CODE REVIEW

**Comment by TheVekter** [comment]
first

I don't like this, it's dumb, it removes the entire point of cult

**Comment by trexter5555** [comment]
so basically this locks constructs to wizard only, kinda sad since they can be a hit or miss for cult

**Comment by trexter5555** [comment]
bring back clock cult

**Comment by zxaber** [comment]
This seems like just team heretics. Eh, I guess, not a fan. Had a few really fun rounds last night with cult that wouldn't have been the same without conversion.

Also, if you're locking constructs to only borg conversions, you should just remove constructs altogether and make the borg cultist-aligned like what clock cult used to do. Being a borg is way more interesting than unga bunga simple mob.

**Comment by tralezab** [comment]
Makes cult too shallow imo. At least heretic progresses and has alt-paths and shit, this REALLY is starting at the maximum cult size you can get roundstart and needing to kill x number of people to win the game. I hate it

**Comment by private-tristan** [comment]
what happens if you're the only competent cultist and the rest of your team just sucks/gets unlucky and dies within 5 minutes of roundstart (I really feel like the only competent cultist half the time)? are you just unable to win? what happens to the people sacrificed? are they round removed because there's no constructs? what purpose do artificers serve now that they can't make shards or constructs, literally just magic missile and healing what few other constructs exist?

won't this also effect wizard's artificer summon+soulshard belt because artificers can't make more shards/constructs for the wizard army?

**Comment by davethwave** [comment]
Best to do a few test merge rounds before a solid decision can be made in my opinion.

**Comment by Iamgoofball** [comment]
If it doesn't go through, then it doesn't go through, but it's better for us to try something new with cult rather than let it sit and rot.

**Comment by tralezab** [comment]
> what happens if you're the only competent cultist and the rest of your team just sucks/gets unlucky and dies within 5 minutes of roundstart (I really feel like the only competent cultist half the time)? are you just unable to win? what happens to the people sacrificed? are they round removed because there's no constructs? what purpose do artificers serve now that they can't make shards or constructs, literally just magic missile and healing what few other constructs exist?
> 
> won't this also effect wizard's artificer summon+soulshard belt because artificers can't make more shards/constructs for the wizard army?
> 
> edit: what happens if one of the target leaves the Z level? is the cult able to reroll their sacrifice target if a sacrifice target decides to spend the entire round on lavaland/exploring space?

As much as I disagree with this direction, this is changing cultists from "might have signed up to be a cultist, might have just been converted with no idea on what to do" to "everyone in the cult is a cultist" so this should improve the quality

**Review by MrMelbert — CHANGES_REQUESTED** [review]

**Comment by ArcaneMusic** [comment]
Do you think this would benefit from more objectives being added for the cult to perform, like different rituals than just summoning narsie?

File: code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm [hunk 1]
```diff
@@ -323,22 +323,17 @@
 		JOB_SECURITY_OFFICER,
 		JOB_WARDEN,
 	)
-	required_candidates = 2
+	required_candidates = 9
 	weight = 3
 	cost = 20
 	requirements = list(100,90,80,60,40,30,10,10,10,10)
```
[the remark this answers is not in the document]
Comment (tralezab) [reply code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm:329]: 9 person cult + 18 sacrifices to summon = at least 27 pop needed

**Comment by SuperNovaa41** [comment]
I like the idea of shaking up cult but this way seems pretty lame.
I think conversion is the best part of cult tbqh, this makes it seems like (as zxaber said aswell) team heretics with different spells.

Another thing you have to think about is how this scales with population. Because 9 cultists on a 30 pop station seems pretty fair but 9 cultists on a 90 pop station would get absolutely stomped.

**Comment by itseasytosee** [comment]
I knew this was a good pr without reading the name

**Comment by Acer202** [comment]
I actually really like this. It makes more sense that an entire cult snuck onto the station beforehand than a cult making people suddenly want to turn into meat lobsters for an eldritch horror.

And

My god

A good way to kill the cheese touch. The stinky hand. The funny grab. GONE. No more one-click victories in a battle against a changeling.

**Comment by Wallemations** [comment]
I agree that cult needs a shakeup but this just feels kinda bland to me to be honest. It just seems like team old-traitor to me, kill a bunch of people then greentext.

**Comment by Palpetinus** [comment]
From a gameplay balance standpoint, cult like this would be extremely overpowered

1. the massive downside of cult is that besides the 1-3 starting cultists, most cultists are uncoordinated bumbling idiots
2. giving 9 people who cannot convert an instant stun is such a bad idea I don't even have to justify it, I only have to say two words "Security" and "Tasers". Cult having the instant stun is only tolerated because they convert so anyone who gets stunned is not round removed
3. stealth would mostly be dead. As soon as cult is found out, either sec powerplays to the high heavens and bombs and brutalizes the cult with heavy weaponry, or NINE people murderbone a shift to death with instant stuns. It's like nukies if nukies were extremely infuriating, even more so than now! 

Cult does need a big shakeup, but this ain't it chief

**Comment by Iamgoofball** [comment]
Yeah I mean I can remove the funny hand

**Comment by CPTANT** [comment]
No no no, don't change cult like this. If you want this premise please implement it as a new antagonist. Tug of war with the core mechanics of an antagonist type is really bad. Remember what happened to clock cult?

**Comment by Mothblocks** [comment]
do you remember what happened to blood cult

**Comment by necromanceranne** [comment]
I'm not entirely convinced this is the right thing to do. I have a few concerns about cult getting BTFO super hard despite the gamer gear, and since you absolutely need three people to accomplish the primary tasks of the role, I anticipate there being a problem of 'you've lost but since you're two dudes left, you're technically still an antag with nothing you can accomplish OTHER than mass murder everyone in revenge'.

I really would rather we didn't add a dead-in-the-water scenario personally. It doesn't need to be fixed via conversion mechanics, but I think allowing something for the remaining cultist to accomplish would be a good idea.

**Comment by 567Turtle** [comment]
I don't like this, it's dumb, it removes the entire point of cult

**Comment by tralezab** [comment]
we should really just readd reebe clock cult

**Comment by Valorium-RB** [comment]
I...actually don't think I would mind this as either a new antagonist or an alternative version of cult that can spawn when it rolls. With the second option, it'd require more investigation by Security into what sort of cult it is and whether to invest in mindshields or in more firearms, or whether to secure cargo at all. I _could_ see it working in either of those two scenarios. As a full replacement? Nah.

**Comment by Mothblocks** [comment]
dont do this to me @tralezab

**Comment by SexmasterWasTaken** [comment]
Goofballs level PR what is this lol, actual joke

**Comment by datorangebottle** [comment]
While I like the idea of repeating numbers( nine cultists, nine sacrifices, etc.), I heavily dislike the gutting and essential removal of constructs.

**Comment by tralezab** [comment]
> dont do this to me @tralezab

we should really just readd reebe clock cult **;)**

**Comment by jlsnow301** [comment]
> we should really just readd reebe clock cult **;)**

[obligatory](https://github.com/BeeStation/BeeStation-Hornet/pull/2124)

**Comment by Iamgoofball** [comment]
@datorangebottle We can't have no conversion and constructs at the same time, constructs function as conversion for nonconvertables. On the upside, we could buff constructs since they're much rarer by virtue of being twisting aura accessible only.

**Comment by 567Turtle** [comment]
Also might I add, theres a reason that there hasn't been a big update to Nar'Sie cult in a long time, because it's perfectly fine and fun as it is. If you have 9 people with some pretty good melee weapons, no stun, no constructs, no backup, no ability to get more cultists, security will pick off each of the 9 one by one, because they're traitors with no uplinks with sharpened knives

**Comment by YakumoChen** [comment]
I don't get it

Whats stopping them from just banding together as a group of 9 and absolutely mobbing the shit out of everyone with 30 force swords and hardstun Bolas in the first 5 minutes of the round?

**Comment by YakumoChen** [comment]
Like imagine being the captain alright

You walk out of your quarters after giving the station a silly name

Then suddenly your office gets hit by an EMP

And 9 traitors with eswords crowbar into your office and kill you

That's what this effectively is

**Comment by itseasytosee** [comment]
Unbiddle Cult

**Comment by Mothblocks** [comment]
> @datorangebottle We can't have no conversion and constructs at the same time, constructs function as conversion for nonconvertables. 

FWIW the rule is not no conversion, it's no conversion that needs a wiki. Constructs that just beat people up is totally in line

**Comment by 567Turtle** [comment]
Tg maintainers and headcoders on their way to merge the worst PR ever that nobody in the player base likes

**Comment by SuperNovaa41** [comment]
I have not seen any maintainer interest in this, both on this PR and in the discord.
Aswell, I really do not like this, it makes cult feel very static. I like conversion, cult's might be a little shitty, but I think it's still probably the best part of cult.

**Comment by Mothblocks** [comment]
this shouldve been closed a while ago for not having a design doc btw

**Pull request closed without merging**